Bomber Guys/map.xml

```diff
@@ -1,10 +1,10 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <map proto="1.4.0">
   <name>Bomber Guys</name>
-  <version>1.0.2</version>
+  <version>1.0.3</version>
   <objective>Blow up your enemy!</objective>
   <rules>
-    <rule>Jumping, Fall damage, Explosion damage by self and ally are disabled.</rule>
+    <rule>Fall damage, Explosion damage by self and ally are disabled.</rule>
     <rule>Don't try to hit the EGG Grenade on the Falling Anvil!</rule>

```

Comment:
```xml
<rule>Fall damage is disabled.</rule>
<rule>You can not damage yourself or your team with TnT</rule>
<rule>Don't throw EGG Grenades on the Falling Anvil!</rule>
```

I think the sentences here need to be rephrased somehow, left a suggestion above, but unsure myself. Not even sure if its necessary to mention fall damage/TnT damage as a rule, but maybe a tip at the start? I think its pretty self explanatory when the map runs :)